spec/helpers/application_helper_spec.rb

```diff
@@ -23,38 +25,43 @@
     before do
       allow(helper).to receive(:controller_name).and_return('my_controller')
       allow(helper).to receive(:action_name).and_return('an_action')
+      allow(helper).to receive(:title)
 
       # So we can simulate what would happen on production
       allow(
         Rails.application.config
       ).to receive(:consider_all_requests_local).and_return(false)
     end
 
-    it 'should call #title with a blank value' do
-      expect(helper).to receive(:title).with('')
+    it 'calls #title with a blank value' do
       helper.fallback_title
+
+      expect(helper).to have_received(:title).with('')
     end
   end
 
   describe '#decorate' do
     before do
       stub_const('FooBar', Class.new)
       stub_const('FooBarDecorator', Class.new(BaseDecorator))
+      allow(FooBarDecorator).to receive(:new)
     end
 
     let(:foobar) { FooBar.new }
 
-    context 'for a specific delegator class' do
+    context 'when for a specific delegator class' do
```

Comment:
I get this is the --autocorrect doing its magic, but some rephrasing has some small defects.
Not a big deal of course.